src/registry_schemas/schemas/conversion.json

```diff
@@ -8,46 +8,53 @@
   "title": "Conversion Filing",
   "properties": {
     "conversion": {
-      "required": [
-        "nameRequest",
-        "offices",
-        "parties",
-        "shareStructure",
-        "contactPoint"
-      ],
       "type": "object",
-      "title": "Conversion Filing",
-      "properties": {
-        "nameRequest": {
-          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
-        },
-        "offices": {
-          "registeredOffice": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-          },
-          "recordsOffice": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-          },
-          "required": [
-            "registeredOffice"
-          ]
-        },
-        "parties": {
-          "type": "array",
-          "items": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
+      "required": [
+          "offices",
+          "parties",
+          "contactPoint"
+        ],
+        "properties": {
+          "business": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/business#/properties/business"
+          },
+          "startDate": {
+            "type": "string",
+            "format": "date"
+          },
+          "nameRequest": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
+          },
+          "offices": {
+            "registeredOffice": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+            },
+            "recordsOffice": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+            },
+            "businessOffice": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+            }
+          },
+          "parties": {
+            "type": "array",
+            "items": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
+            }
+          },
+          "shareStructure": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/share_structure"
+          },
+          "contactPoint": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint"
+          },
+          "incorporationAgreement": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/agreementType"
+          },
+          "courtOrder": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/court_order#/properties/courtOrder"
           }
```

Comment:
I think the NAICS object should be in here, so that staff can update the nature of business (naics desc but not naics code).

Reply:
Oh wait... NAICS is in [business schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/business.json).

I'm resolving this comment.